lib/elixir/lib/code.ex

```diff
@@ -32,16 +62,23 @@ defmodule Code do
   ## Examples
 
       # Load EEx test code, unload file, check for functions still available
-      Code.load_file("../eex/test/eex_test.exs")
+      Code.require_file("../eex/test/eex_test.exs")
 
-      Code.unload_files(Code.loaded_files())
+      Code.unrequire_files(Code.required_files())
       function_exported?(EExTest.Compiled, :before_compile, 0)
       #=> true
 
   """
-  @spec unload_files([binary]) :: :ok
+  @spec unrequire_files([binary]) :: :ok
+  @since "1.7.0"
+  def unrequire_files(files) do
+    :elixir_code_server.cast({:unrequire_files, files})
+  end
+
+  # TODO: Deprecate me on 1.9
+  @doc false
   def unload_files(files) do
-    :elixir_code_server.cast({:unload_files, files})
+    :elixir_code_server.cast({:unrequire_files, files})
```

Comment:
As we no longer test this function can we call `unrequire_files/1` instead here, as a change to unrequire could break this unknowingly.